src/libraries/Microsoft.PowerFx.Core/Texl/Builtins/Summarize.cs

```diff
@@ -124,9 +124,16 @@ public override bool CheckTypes(CheckTypesContext context, TexlNode[] args, DTyp
                 // All args starting at index 1 need to be identifiers or aggregate functions (wrapped in AsNode node).
                 switch (arg)
                 {
-                    case AsNode nameNode:
+                    case AsNode asNode:
+
+                        if (asNode.Left is FirstNameNode || asNode.Left is DottedNameNode)

```

Comment:
Double checking that we want to also fail on DottedName node?

Reply:
Turns out we shouldn't fail. Thanks.